blueprint/api.apib

```diff
@@ -279,6 +279,165 @@ You can list all comments or retrieve individual comments, as well as create and
 
     + Attributes (Unprocessable Entity Response)
 
+# Group Donation Goals
+
+This resource describes one of any number of donation goals a project might have. A project cannot create a donation goal until they have started accepting payment with Stripe.
+
+## Donation Goals [/donation-goals]
+
+### Create a donation goal [POST]
+
+To create a donation goal the user must be a site admin or the owner of the referenced project.
+
++ Parameters
+
+    + id (number) - The `id` of a donation goal.
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer {token}
+
++ Response 201 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Donation Goal Create Request)
+
++ Response 401 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (JSON Web Token Invalid Response)
+
++ Response 403 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Forbidden Response)
+
++ Response 404 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Record Not Found Response)
+
++ Response 422 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Unprocessable Entity Response)
+
+### List all donation goals [GET]
+
++ Request
+
+  + Headers
+
+            Accept: application/vnd.api+json
+
++ Response 200 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Donation Goals Response)
+
+### Filter donation goals by id [GET /donation-goals{?filter[id]}]
+
++ Parameters
+
+    + `filter[id]`: `1,2,3` (string, required) - Comma separated string of `ids` to filter by.
+
+        Technically, the parameter is `filter: { id: '1,2,3' }`, which ends up being serialized into `filter[id]`. However, syntax constraints do not allow us to define it in the documentation this way.
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+
++ Response 200 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Donation Goals Response)
+
++ Response 404 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Record Not Found Response)
+
+## Donation Goal [/donation-goals/{id}]
+
+### Retrieve a donation goal [GET]
+
++ Request
+
+    + Headers
+
+          Accept: application/vnd.api+json
+
++ Response 200 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Donation Goal Response)
+
++ Response 404 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Record Not Found Response)
+
+### Update a donation goal [PATCH]
+
+To update a donation goal the user must be a site admin or the owner of the referenced project.
+
++ Parameters
+
+    + id (number) - The `id` of a donation goal.
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer {token}
+
++ Response 200 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Donation Goal Update Request)
```

Comment:
Similar to the create request above, you're defining the response as a request. Apply the same pattern you see with the create request/response to the request/response here.